diff -Nuar a/vboxvideo/vbox_fb.c b/vboxvideo/vbox_fb.c
--- a/vboxvideo/vbox_fb.c	2026-08-20 13:48:18.000000000 +0300
+++ b/vboxvideo/vbox_fb.c	2026-08-24 15:53:38.422472527 +0300
@@ -333,8 +333,8 @@
 #if RTLNX_VER_MIN(6,19,0) || RTLNX_RHEL_RANGE(9,9, 9,99)
 	info = helper->info;
 #elif RTLNX_VER_MIN(6,2,0) || RTLNX_RHEL_RANGE(8,9, 8,99) || RTLNX_RHEL_RANGE(9,3, 9,99)
-	info = drm_fb_helper_alloc_info(helper);
 #else
+	info = drm_fb_helper_alloc_info(helper);
 	info = drm_fb_helper_alloc_fbi(helper);
 #endif
 	if (IS_ERR(info))
